bluesky_adaptive/per_event.py

```diff
@@ -81,6 +82,9 @@ def callback(name, doc):
             except NoRecommendation:
                 # no recommendation
                 queue.put(None)
+            except RunEngineControlException as e:
```

Comment:
Handle the `except Exepction` case here as well so that it get re-raised correctly in L173 rather than being raised in which ever stack this callback is running in.

Reply:
Added generic exception pass through to plan here https://github.com/bluesky/bluesky-adaptive/pull/29/commits/8b7005270be5ecab4a787e1f07b5e745058d5e32